episode-6.msrt

```diff
@@ -183,8 +182,7 @@
 37
 00:02:32,220 --> 00:02:35,220
 [ru] ...гостья, из страны заходящих лун...
-[en] [Version 1]...comes to us from the lands of the setting moons...
-[en] [Version 2]...from the lands of the setting moons...
+[en] ...comes to us from the lands of the setting moons...
```

Comment:
Agree. It is not yet clear which version will get into final editing, though. But I believe we can cut "comes to us" from audio if full phrase won't fit.